spec/requests/users_spec.rb

```diff
@@ -0,0 +1,130 @@
+ require 'rails_helper'
+
+# This spec was generated by rspec-rails when you ran the scaffold generator.
```

Comment:
I was amazed when I ran `rspec` and saw > 100 tests! Did you write most of them or are they generated with scaffold?